src/style.css

```diff
@@ -1,3 +1,75 @@
 body {
   margin: 0;
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+  margin: 0;
+  justify-content: space-between;
+  background-color: rgb(255, 255, 255);
+}
+
+ul.nav__list {
+  background-color: rgb(255, 255, 255);
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+li.nav__item {
+  display: flex;
+  margin: 0;
+}
+
+a.nav__link {
+  outline: none;
+  text-decoration: none;
+  color: rgba(0, 0, 0, 1);
+  background-color: rgb(255, 255, 255);
+}
+
+.nav a {
+  line-height: 60px;
+  height: 60px;
+  margin: 0;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+a:hover {
+  color: rgba(0, 172, 220, 1);
```

Comment:
* Consider creating variables for repeating colors
* Consider using hex values, because it's more declarative